src/@components/@common/textLength.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+
+interface TextLengthProps {
+  inputLength: number;
+  limit: number;
+  font: string;
+}
+
+export default function TextLength(props: TextLengthProps) {
+  const { inputLength, limit, font } = props;
+
+  return (
+    <Container font={font}>
+      <InputLength inputLength={inputLength}>{inputLength}</InputLength>
+      <Limit>/{limit}</Limit>
+    </Container>
+  );
+}
+
+const Container = styled.div<{ font: string }>`
+  display: flex;
+  ${(props) => props.font};
+`;
+
+const InputLength = styled.p<{ inputLength: number }>`
+  color: ${(props) => (props.inputLength > 0 ? ({ theme }) => theme.colors.white : ({ theme }) => theme.colors.gray3)};
```

Comment:
```suggestion
  color: ${({inputLength, theme})=> inputLength > 0 ? theme.colors.white : theme.colors.gray3}
```
이렇게두 쓸 수 있는데, 해당 방식으로 작성한 이유가 궁금해요!!

Reply:
무지함이랄까요..?